Content.Shared/Chameleon/SharedChameleonSystem.cs

```diff
@@ -19,46 +15,94 @@ public override void Initialize()
         SubscribeLocalEvent<SharedChameleonComponent, ComponentGetState>(OnChameleonGetState);
         SubscribeLocalEvent<SharedChameleonComponent, ComponentHandleState>(OnChameleonHandlesState);
         SubscribeLocalEvent<SharedChameleonComponent, MoveEvent>(OnMove);
+        SubscribeLocalEvent<SharedChameleonComponent, EntityPausedEvent>(OnPaused);
+        SubscribeLocalEvent<SharedChameleonComponent, ComponentInit>(OnInit);
```

Comment:
Accounting for paused entities makes this all more convoluted than it has to be. I just made  last-modified nullable, where null implies that no passive visibility changes should be applied.